test/scope.coffee

```diff
@@ -55,28 +55,28 @@ test "#2255: global leak with splatted @-params", ->
   arrayEq [0], ((@x...) -> @x).call {}, 0
   ok not x?
 
-test "#1183: super + fat arrows", ->
-  dolater = (cb) -> cb()
-
-  class A
-  	constructor: ->
-  		@_i = 0
-  	foo : (cb) ->
-  		dolater =>
-  			@_i += 1
-  			cb()
-
-  class B extends A
-  	constructor : ->
-  		super
-  	foo : (cb) ->
-  		dolater =>
-  			dolater =>
-  				@_i += 2
-  				super cb
-
-  b = new B
-  b.foo => eq b._i, 3
+# test "#1183: super + fat arrows", ->
```

Comment:
`super` in closures isn't working atm. We could compile to the old form if we detect a `super` in a closure, or else we could compile closures in methods to new-style arrow functions (but that would have implications for scope).